### PR TITLE
Update sellto.ts so that qty is optional

### PR DESCRIPTION
**Pull request by BrandonEnglish848**
If the quantity being sold is not specified in the original command, the bot will find the item's id and search the user's bank and then set the quantity being sold to that number. 
I also normalized a bit of your layout discrepancies between the if statements.
✅ I have tested my changes

### CODE REVIEW

File: src/commands/Minion/sellto.ts [hunk 1]
```diff
@@ -30,29 +30,44 @@ export default class extends BotCommand {
...
+		}
+		if (buyerMember.user.isIronman) {
+			throw `Iron players can't be sold items.`;
+		}
+		if (buyerMember.user.id === msg.author.id) {
+			throw `You can't trade yourself.`;
+		}
+		if (buyerMember.user.bot) {
+			throw `You can't trade a bot.`;
+		}
```
Comment (gc) [inline src/commands/Minion/sellto.ts:46]: Why did you change all these?

File: src/commands/Minion/sellto.ts [hunk 2]
```diff
@@ -30,29 +30,44 @@ export default class extends BotCommand {
...
 		if (buyerMember.user.isBusy) {
 			throw `That user is busy right now.`;
 		}
-
```
Comment (gc) [inline src/commands/Minion/sellto.ts:42]: Why remove a random empty line?

Comment (BrandonEnglish848) [reply src/commands/Minion/sellto.ts:42]: Because ESLint told me to.

File: src/commands/Minion/sellto.ts (hunk 1 above)
Comment (BrandonEnglish848) [reply src/commands/Minion/sellto.ts:46]: Because ESLint told me to.

**Comment by BrandonEnglish848** [comment]
I have not tested this, and will be messaging in the dev channel

**Comment by BrandonEnglish848** [comment]
### Description:

The Sellto command can have more versatility if the users have already agreed on a price, and are going to sell entire stacks of items. It's much easier for one user to sell a whole stack rather than selling small chucks or have to type out specific quantities.

### Changes:

* Generally Utilized the Sell and Sellto commands
* Changed the run parameters to where quantity is optional.
* Did NOT change the order in which the parameters are passed into the run command so that users who are already familiar with the command will not be confused with the change.

- [✔️] I have tested all my changes thoroughly.

**Comment by Lajnux** [comment]
We going over to slashcommands and a system to make qty optional is now implemented.
